README.md

```diff
@@ -6,7 +6,7 @@
 
 ![drawing](script-languages-container-logo.png)
 
-This project contains script-language containers for the user-defined functions (UDF's) that can be used in the EXASOL database (version 6.2.0 or later). Using script-language containers, user can extend the UDF's with additional libraries and tools, such as Python or R packages and their dependencies.
+This project contains script-language containers for the user-defined functions (UDF's) that can be used in the EXASOL database (version 7.0.0 or later). Using script-language containers, user can extend the UDF's with additional libraries and tools, such as Python or R packages and their dependencies.
```

Comment:
I was not sure if I should update this because actually the containers still work with 6.2.0 But as we don't support this version anymore I find it better to switch 7.0.0 in order to avoid possible annoying tickets.

Reply:
yes, no mentation of 6.2 anymore